client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-item.jsx

```diff
@@ -29,9 +29,41 @@ const EmailForwardingItem = React.createClass( {
 			this.recordEvent( 'deleteClick', domain, mailbox, forward_address, ! Boolean( error ) );
 
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'Failed to delete email forwarding record. Please try again or contact customer support if error persists.' ) );
```

Comment:
We usually say "contact support if it persists." and have a link to the support url.